pycode/epidemiology/epidata/getRKIData.py

```diff
@@ -41,7 +41,7 @@
 from epidemiology.epidata import geoModificationGermany as geoger
 
 
-def check_for_completeness(df):
+def check_for_completeness(df, merge_berlin = False, merge_eisenach = False):
```

Comment:
If I remember correctly, Eisenach is merged in current RKI data so the default argument there should be true?!